test/logout.tests.js

```diff
@@ -152,4 +152,46 @@ describe('logout route', function() {
       });
     });
   });
-});
\ No newline at end of file
+
+  describe('logout with custom path', function() {
+    let baseUrl;
+    let currentSession;
+    const jar = request.jar();
+
+    before(async function() {
+      const middleware = auth({
+        idpLogout: false,
+        clientID: '__test_client_id__',
+        baseURL: 'https://example.org/foo',
+        issuerBaseURL: 'https://test.auth0.com',
+        appSession: {
+          secret: '__test_secret__',
+          cookiePath: '/foo'
+        },
+        required: false
+      });
+      baseUrl = (await server.create(middleware, null, '/foo')) + '/foo';
+      await request.post({
+        uri: '/session',
+        json: {
+          openidTokens: {
+            id_token: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'
+          },
+          claims: {}
+        },
+        baseUrl, jar
+      });
```

Comment:
I hear you - but I'm loosely following the style of the other tests. I don't really like the way be use `before` here because it makes the order of the tests important. But, at least this way we're setting up the `should clear the session` test in `before` rather than in the `should populate the session` test.